src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <main class="product-cards">
+      <header class="nav">
+        <img
+          src="/images/logo.png"
+          alt="icon"
+          class="nav_image"
+        >
+        <nav>
+          <ul class="nav_list">
+            <li class="nav_item">
+              <a class="nav_link is-active" href="#">
+                APPLE
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link" href="#">
+                SAMSUNG
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link" href="#">
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link nav_link--color_blue"             data-qa="nav-hover"
+              href="#">
+                LAPTOPS & COMPUTERS
+              </a>
```

Comment:
Pay attention on indents
```suggestion
              <a 
                class="nav_link nav_link--color_blue"
                data-qa="nav-hover"
                href="#"
              >
                LAPTOPS & COMPUTERS
              </a>
```